doc/doc-site/docs/off-chain/python-sdk.md

```diff
@@ -1,3 +1,262 @@
+
+### Running the Example Script ("Trading From Scratch")
+
+You're recommended to run a local Aptos node and faucet first; this prevents any rate-limitting issues from preventing your progress. Install the Aptos CLI & run your local node/faucet:
+
+```bash
+brew install aptos # only if necessary
+mkdir aptos && cd aptos
+aptos node run-local-testnet --with-faucet
+```
+
+In another terminal, run the following:
+
+```bash
+export APTOS_NODE_URL=http://0.0.0.0:8080/v1
+export APTOS_FAUCET_URL=http://0.0.0.0:8081
+```
+
+You'll also have the option of entering these as prompts of the script, but the environment variable is preferred because it's easier to run multiple times. It's time to deploy our own Econia Faucet to the local chain:
+
+```bash
+git clone https://github.com/econia-labs/econia.git # only if necessary
+cd ./econia/src/move/faucet
+aptos init --profile econia_faucet_deploy # enter "local" for the chain
+export FAUCET_ADDR=<ADDR-FROM-ABOVE>
+# deploy the faucet (all one command)
+aptos move publish \
+        --named-addresses econia_faucet=$FAUCET_ADDR \
+        --profile econia_faucet_deploy \
+        --assume-yes
+```
+
+We also need to deploy an Econia exchange:
+```bash
+cd ./econia/src/move/econia
+aptos init --profile econia_exchange_deploy # enter "local" for the chain
+export ECONIA_ADDR=<ADDR-FROM-ABOVE>
+# deploy the faucet (all one command)
+aptos move publish \
+        --override-size-check \
+        --included-artifacts none \
+        --named-addresses econia=$ECONIA_ADDR \
+        --profile econia_exchange_deploy \
+        --assume-yes
+```
+
+It's to run the script! Setting our environment variables will have cleared the initial setup prompts for us. In order to run, install Poetry then install dependencies and run the script:
+
+```bash
+curl -sSL https://install.python-poetry.org | python3 - # only if necessary
+cd ./econia/src/python/sdk && poetry install # only if necessary
+poetry run trade # we're off to the races!
+```
+
+### Understanding the Example Script
+
+#### Step #1: Set-up the Market
+
+```
+Press enter to initialize (or obtain) the market.
+Market does not exist yet, creating one...
+EVENT SUMMARY: MarketRegistrationEvent
+  * Base Type (unit of lots): 0x...::test_eth::TestETH
+  * Quote Type (unit of ticks): 0x...::test_usdc::TestUSDC
+Market ID: 1
+TRANSACTIONS EXECUTED (first-to-last):
+  * Create a new market: 0x544c1535d9634bb4d4f5ee5b74ae798e45e5b1a0e54bfea060026e5ce0163dd9
+There are no open orders on this market right now.
+```
+
+In this case, the market did not exist for the pair of faucet currencies (`TestEth` and `TestUSDC`) that we deployed. Only one market can exist per unique configuration so once it's created here, it won't be created again in future runs. Instead it'll be detected as existing via a view function, and then re-used from thereon. There don't appear to be any open orders on this brand-new market which makes a lot of sense!
```

Comment:
Can you add a comment here saying how the long hexstring is the transaction hash, and that it can be looked up on an explorer?